packages/google_adsense/test/adsense_apis_test.dart

```diff
@@ -0,0 +1,21 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+@TestOn('vm')
```

Comment:
This code should never be executed in the `vm`, so this test can possibly be removed.